### DISCUSSION TITLE
Welcome to potato Discussions!

### DISCUSSION THREAD

**Discussion by mbridak** [mbridak/potato#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by adammelancon** [comment]
Found your app when searching github for the POTA API link to see what people were using it for.
There's a bunch about my ham adventures on my instagram @KD5QZG  [http://kd5qzg.com](http://kd5qzg.com)
Also a pretty [large bio on QRZ ](https://www.qrz.com/db/KD5QZG)

Great app! Glad I got to help out a bit!  This was my first ever pull request that merged into someone else's code.  Feels great to be a small part of something.

PS: I don't know how to make the "issue" I added about checking off as contacted as an "enhancement" or "Feature Request", so it doesn't look like something is wrong with the app.

**Reply by mbridak** [reply]
Hello Sir!
Thanks for bringing up the issue and submitting the PR. I don't use Windows myself and would have never had known.

Re: Feature request.
Yes it does seem like a worthwhile feature. I think I will implement it as a right-click option, as soon as I figure out how to differentiate between a left and right click...
